crates/matrix-sdk-sled/src/state_store.rs

```diff
@@ -150,8 +174,129 @@ const TIMELINE_METADATA: &str = "timeline-metadata";
 #[cfg(feature = "experimental-timeline")]
 const TIMELINE: &str = "timeline";
 
+const ALL_DB_STORES: &[&str] = &[
+    ACCOUNT_DATA,
+    SYNC_TOKEN,
+    DISPLAY_NAME,
+    INVITED_USER_ID,
+    JOINED_USER_ID,
+    MEDIA,
+    MEMBER,
+    PRESENCE,
+    PROFILE,
+    ROOM_ACCOUNT_DATA,
+    #[cfg(feature = "experimental-timeline")]
+    ROOM_EVENT_ID_POSITION,
+    ROOM_EVENT_RECEIPT,
+    ROOM_INFO,
+    ROOM_STATE,
+    ROOM_USER_RECEIPT,
+    ROOM,
+    STRIPPED_INVITED_USER_ID,
```

Comment:
```suggestion
    SESSION,
    STRIPPED_INVITED_USER_ID,
```
Since it was changed from plain to encrypted, we also need to drop it.